container/flash_iot.sh

```diff
@@ -41,7 +41,6 @@ cleanup () {
 	exit_code=$?
 	umount $balena_image_boot_mnt > /dev/null 2>&1 || true
 	losetup -d $balena_image_loop_dev > /dev/null 2>&1 || true
-	losetup -D
```

Comment:
Can you leave a detailed git log message for this commit explaining why this was removed?